joy_mode_handler:
  ros__parameters:
    estop_button: 5 # R1 Shoulder button
    auto_button: 0 # A button
